rotkehlchen/types.py

```diff
@@ -964,7 +964,6 @@ class CostBasisMethod(SerializableEnumNameMixin):
     ACB = auto()
 
 
-ANY_BLOCKCHAIN_ADDRESSBOOK_VALUE: Final = 'NONE'  # blockchain value used to mark in the DB that the address entry is valid for any blockchain TODO: remove  # noqa: E501
```

Comment:
the one below is TODO remove in another PR?

Reply:
no, that todo sholdn't be there :( it is the thing that we use now instead of NONE